articles/143_legacy_interface_definition.md

```diff
@@ -253,7 +253,7 @@ Code generation uses the generated file.
 | -------- | ------------------ |
 | bool     | boolean            |
 | byte     | octet              |
-| char     | char               |
+| char     | short              |
```

Comment:
would an `octet` make more sense than a `short` in this case ?

Reply:
I think `short` is a better fit than `octet`. IDL 4.2 section `7.4.1.4.3` says `Octet literals have integer value in the range 0…255`. `char` is an integer in range [-128, 27] in ros 2 .msg files https://github.com/ros2/design/pull/190#discussion_r220283197

Reply:
It's still unclear why it is treated differently than the `int8` type.

My understanding was that they both fit in 8-bit and can represent signed values, so I'm not sure why we should use a 16-bit type to store them

Reply:
Got the int8 types mixed up. It should be fixed in e7c99eafecd86ca8f41f106047177eb56902a64d